src/Models/ModelInstantiatorTest.php

```diff
@@ -21,17 +22,24 @@ public function testArrayToDataObject(): void
             'firstName'    => 'Peter',
 
             // testProperty exists in the class but does not have a Column attribute.
-            'testProperty' => 'test',
+            'testProperty' => true,
 
             // fakeProperty does not exist in the class.
             'fakeProperty' => 'faker',
         ];
 
-        // Convert the array to a data object.
-        $data_object = $this->model_instantiator->arrayToDataObject(
-            $array,
-            ModelInstantiatorTestObject::class
-        );
+        try
```

Comment:
Is the try/catch necessary? We don't expect the function to throw an exception, right?

Reply:
Works for me.